src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -202,20 +205,27 @@ public void info(CommandSender sender) {
 			tellraw.addText("   &7&oHover for more Info");
 			StringBuilder hover = new StringBuilder();
 			int hidden = 0;
-			
-			for (Map.Entry<String, Integer> entry: machineCount.entrySet()) {
-				long timings = machineTimings.get(entry.getKey());
-				if (timings > 0) hover.append("\n&c").append(entry.getKey()).append(" - ").append(entry.getValue()).append("x &7(").append(timings).append("ms)");
-				else hidden++;
+
+			Map<String, Long> timings = machineCount.entrySet().stream()
+					.sorted(Comparator.comparingLong(machineTimings::get).reversed())
+					.collect(Collectors.toMap(Map.Entry::getKey, machineTimings::get, (e1, e2) -> e1, LinkedHashMap::new));
```

Comment:
Why do you use machineTimings:: get as the value?
Aren't you discarding the actual values from the map there...